test/test_views.py

```diff
@@ -31,11 +31,15 @@ def _create_storage(self):
     def _create_blogging_engine(self):
         return BloggingEngine(self.app, self.storage)
 
-    def setUp(self):
+    def other_config(self):
+        pass
+
+    def setUp(self, **config):
```

Comment:
You perhaps forgot to change the signature of this function back to `setUp(self)`.